### DISCUSSION TITLE
Worker primary ip, debug purpose

### DISCUSSION THREAD

**Discussion by Roxedus** [hcloud-talos/terraform-hcloud-talos#69] [General]
I was looking into the resources this creates, and wondered what debug purposes always having a primary ip for a worker node fills?
This is done deliberately, as outlined by the comment https://github.com/hcloud-talos/terraform-hcloud-talos/blob/main/network.tf#L61

**Comment by mrclrchtr** [comment]
I did this to be able to run and test terraform without having to create CPs. This sped up development at the time. In reality, it can't happen that no CP is created. That's why I left it in.

**Reply by Roxedus** [reply]
That makes sense, however the logic is the same for workers. I would assume the same debug claim were supposed to apply there, but I don't see why it always should create a primary worker ip on a single node setup.

**Reply by mrclrchtr** [reply]
Oh, you're right... that's a mistake. It shouldn't matter for workers. I'll have a look at it soon.

**Reply by mrclrchtr** [reply]
Created https://github.com/hcloud-talos/terraform-hcloud-talos/issues/70
